src/components/TextInput/TextInput.tsx

```diff
@@ -116,9 +116,7 @@ export const TextInput = chakra(
       const finalInvalidText = invalidText
         ? invalidText
         : "There is an error related to this field.";
-      let footnote: HelperErrorTextType = isInvalid
-        ? finalInvalidText
-        : helperText;
+      let footnote = isInvalid ? finalInvalidText : helperText;
```

Comment:
`footnote` actually does need to be typed but it's not throwing an error since the "strict" flag is still set to "false". This was a harder update to do because `HelperErrorText` expects it to be `string | JSX.Element` but `helperText` can actually be undefined in this component (and other similar components). So this is something that needs a refactor but that was more involved and left it for a later PR.

Reply:
OK, thanks for the explanation.